docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -404,6 +399,51 @@ and BatchScanner passing in the name of the context, app1 in the example above.
 iterators to load classes from the locations defined by the context. Passing the context name to the scanners allows you to override the table setting
 to load only scan time iterators from a different location. 
 
+=== Vendor-specific configuration
```

Comment:
I think this info would better on the website, where it can be easily updated and appended to.

We could create the web page now w/o any links to it on the website .. and a disclaimer about the info being for 2.0.0-SNAP.  Then 2.0.0-SNAP docs could point to it.